src/schemas/vis-state-schema.js

```diff
@@ -76,7 +77,8 @@ class DimensionFieldSchemaV0 extends Schema {
     };
   }
 
-  load(field, config, accumulated) {
+  load(field, parents, accumulated) {
+    const [config] = parents.slice(-1);
```

Comment:
could parents be empty?

If empty you may have issues when you access config later on line 83

Reply:
This is been called recursively from `VisStateSchema -> LayerSchema -> LayerConfigSchema` `parents` contains all the parent node `[visState, layer, config]`, it should never be empty. If it is, it means something is wrong, and I would rather the error been thrown instead of a silent pass.